ext/storage_sources/s3_store/s3_log_system.cpp

```diff
@@ -54,8 +54,14 @@ S3LogSystem::LogAwsMessage(const char *tag, const std::string &message) const
 void
 S3LogSystem::LogVerboseMessage(int32_t verbosityLevel, const std::string &message)
 {
-    if (verbosityLevel <= _wtVerbosityLevel)
-        _wtApi->err_printf(_wtApi, NULL, "%s", message.c_str());
+    if (verbosityLevel <= _wtVerbosityLevel) {
+        /* Use err_printf for error and warning messages. */
```

Comment:
I will prefer the two comments be merged into one at the start of the if block at line 58.